include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -139,9 +143,16 @@ struct gap
     {
         return value >= rhs.value;
     }
-    //!@}
+    //!\}
+
+    //!\privatesection
+    //!\brief The data member.
+    static constexpr rank_type value = 0;
```

Comment:
At that point in time, we thought that all alphabets should have a global accessible `value`.